client/signup/steps/new-user-survey/components/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { Card, FormLabel } from '@automattic/components';
+import styled from '@emotion/styled';
+import FormTextInput from 'calypso/components/forms/form-text-input';
+
+export type Viewport = 'desktop' | 'mobile' | 'tablet';
+type PropsWithViewport = { currentViewport: Viewport };
+export const SurveyFormContainer = styled.div`
+	margin: 0
+		${ ( { currentViewport }: PropsWithViewport ) =>
+			currentViewport === 'desktop' ? 'auto' : '10px' };
+`;
+
+export const StyledCard = styled( Card )`
+	margin: 0 auto;
+	padding: 25px 30px;
+	display: flex;
+	justify-content: center;
+	width: ${ ( props: PropsWithViewport ) =>
+		props.currentViewport === 'mobile' ? '100%' : 'fit-content' };
+	min-width: ${ ( props: PropsWithViewport ) =>
+		props.currentViewport === 'mobile' ? 'unset' : '459px' };
+	&&&&& {
+		margin-bottom: 15px;
+	}
```

Comment:
To avoid the `&&&&&` here, perhaps the margin could be applied to the `SurveyFormContainer`:

https://github.com/Automattic/wp-calypso/blob/11005bebee12a168feeddcbfc762561d9a08e630/client/signup/steps/new-user-survey/components/index.tsx#L7-L11